Triage comment updates (#7216)
microsoft/pxt-arcade: .github/workflows/triage.md

---

## Recommendation for Triage

> **This issue appears to belong in a different repository.**

Reviewer: This is a bit nitpicky and might be better decided with user testing, but I think even this wording might be a bit forward.
```suggestion
**This issue might be better filed in a different repository.**
```

---

> 

Author: added
